test/unit_tests/apps/teachers-digital-platform/js/survey/grade-level-page-spec.js

```diff
@@ -1,36 +1,50 @@
 import { jest } from '@jest/globals';
-import surveys from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/tdp-surveys.js';
-import Cookies from 'js-cookie';
 import {
   ANSWERS_SESS_KEY,
   RESULT_COOKIE,
   SURVEY_COOKIE,
 } from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/survey/config.js';
-import * as modals from '../../../../../../cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js';
 
 const HTML_SNIPPET = `
 <div data-tdp-page="grade-level">
 </div>
 `;
 
-xdescribe('The TDP survey grade-level page', () => {
-  beforeEach(() => {
+describe('The TDP survey grade-level page', () => {
+  let surveys;
+  const remove = jest.fn();
```

Comment:
To make the point where this is called a literal less generic, could this be named `cookieRemove`? Then in the unstable mock it'd be `remove: cookieRemove,` and the mock checks further down would be `expect(cookieRemove.mock.calls[0][0]).toEqual(RESULT_COOKIE);`.

Reply:
Done in 0522503